db/seeds.rb

```diff
@@ -93,13 +112,20 @@
                   tenant_id: bridge_builders1.id
                   )
 
+timmys_vaccines_nigeria.photos << Photo.create!(
+                  image: File.new( "#{Rails.root}/app/assets/images/timmys_vaccines.jpg")
```

Comment:
Line is too long. [89/80]<br>Indent the first parameter one step more than the previous line.<br>Space inside parentheses detected.